cmd/buildah/run.go

```diff
@@ -78,25 +65,19 @@ func runCmd(c *cli.Context) error {
 		return errors.Wrapf(err, "error reading build container %q", name)
 	}
 
-	hostname := ""
-	if c.IsSet("hostname") {
-		hostname = c.String("hostname")
-	}
 	options := buildah.RunOptions{
-		Hostname: hostname,
-		Runtime:  runtime,
-		Args:     flags,
+		Hostname: c.String("hostname"),
+		Runtime:  c.String("runtime"),
+		Args:     c.StringSlice("runtime-flag"),
 	}
 
-	if c.IsSet("tty") {
-		if c.Bool("tty") {
-			options.Terminal = buildah.WithTerminal
-		} else {
-			options.Terminal = buildah.WithoutTerminal
-		}
+	if c.Bool("tty") {
+		options.Terminal = buildah.WithTerminal
+	} else {
+		options.Terminal = buildah.WithoutTerminal
```

Comment:
Six of one, half dozen of another.  Perhaps:
options.Terminal = buildah.WithoutTerminal
if c.Bool("tty") {
    opitions.Terminal = buildah.WithTerminal
}

Reply:
Well we have a lot of code that does this method.  From a anal performance point of view,  The current code is better.  From the total lines of code poit of view, yours is better.  :^)